packages/main/src/DateComponentBase.js

```diff
@@ -97,6 +108,10 @@ class DateComponentBase extends UI5Element {
 		return this.primaryCalendarType || getCalendarType() || localeData.getPreferredCalendarType();
 	}
 
+	get _secondaryCalendarType() {
+		return this.secondaryCalendarType;
```

Comment:
This getter is not needed, just use the property

Reply:
Done